index.js

```diff
@@ -0,0 +1,96 @@
+const hamburger = document.querySelector('.hamburger');
+const navMenu = document.querySelector('.nav-mobile2');
+
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  navMenu.classList.toggle('active');
+});
+
+document.querySelectorAll('.links').forEach((n) => n
+  .addEventListener('click', () => {
+    hamburger.classList.remove('active');
+    navMenu.classList.remove('active');
+  }));
+
+const container = document.querySelector('.container-speakers');
+const speakers = document.createElement('div');
+container.appendChild(speakers);
+speakers.innerHTML = `
+  <div class="speakers-container">
+  <div class="hrtittle">
+  <h2>Featured speakers</h2>
+  <hr>
+</div>
+<div class="speakers-align">
+<section class="speakers-desk">
+  <div class="speakers-left">
+  <div class="speaker1">
+    <div class="img1">
+      <img src="images/christiana2.PNG" alt="">
+    </div>
+    <div class="bio1">
+      <h4>Christiana Figueres</h4>
```

Comment:
- [ ] The speaker section should be created dynamically, not as an HTML literal template. each speaker should be stored in an array of objects with details, then you should loop through the array and display the section dynamically as we did with the cards in the portfolio projects.